### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by elfenars**
[Review on Reviewable](https://reviewable.io/reviews/elfenars/reviewninja-welcome/1)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/elfenars/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (elfenars) [inline ReadMe.md]: test

**Comment by elfenars** [comment]
!fix test

File: ReadMe.md (hunk 1 above)
Comment (elfenars) [inline ReadMe.md]: !fix

Comment (elfenars) [inline ReadMe.md]: !fixed

**Pull request merged**